Sources/TarkaUI/Components/Base Components/IconButton/TUIIconButton.swift

```diff
@@ -77,20 +76,20 @@ public struct TUIIconButton: View, Identifiable {
         height: iconSize.height
       )
       .clipped()
-      .foregroundColor(iconColor ?? style.defaultIconColor)
+      .foregroundColor(iconColor ?? style.inputStyle.foreground)
   }
   
   struct TUIIconButtonStyle: ButtonStyle {
+    let style: Style
     let buttonSize: CGSize
-    let backgroundColor: Color
-    let borderColor: Color
     let isDisabled: Bool
     
     func makeBody(configuration: Configuration) -> some View {
       configuration.label
         .frame(width: buttonSize.width, height: buttonSize.height)
-        .background(backgroundColor)
-        .border(Circle(), width: 1.5, color: borderColor)
+        .background(style.inputStyle.background)
+        .border(Circle(), width: configuration.isPressed ? 1.5 : 0,
```

Comment:
I dont think we should use fractional points

Reply:
@arvindhsukumar - In itself it has

<img width="234" alt="Screenshot 2024-03-25 at 17 00 17" src="https://github.com/tarkalabs/tarka-ui-kit-ios/assets/102943217/40894024-d925-40f2-93f6-81689264eef1">